### PR TITLE
Core edit

### PR DESCRIPTION
**Pull request by moshekruger**

### CODE REVIEW

File: docs/python-api/how-to/track.rst [hunk 1]
```diff
@@ -217,24 +217,19 @@ As a result you will not send sources to Neptune, so they will not be available
...
-   :target: ../../_static/images/notebooks/buttons_03_1.png
-   :alt: image
-
-.. note:: You can use **Upload** as many times as you want.
+- **Neptune**: For modifying configuration.
+- **Upload**: For creating a new checkpoint in Neptune.
+- **Download**: For downloading a specific checkpoint from Neptune.
+- **Activate**: Associates experiments you will be running with this Notebook.
 
+Click **Upload** whenever you want to create new checkpoint in Neptune. As confirmation, Neptune displays a notification with a link.
```
Comment (aniezurawski) [inline docs/python-api/how-to/track.rst]: Moreover, if "Activate" button was clicked, checkpoints will be created automatically after every experiment creation.

File: docs/learn-about-neptune/core-concepts.rst [hunk 2]
```diff
@@ -1,116 +1,164 @@
...
 
 .. _core-concepts_organization-types:
 
-Organization is a way to centrally manage projects, users (and billing). Neptune has two organization types.
+Neptune has two organization types: Individual and Team
+
+Individual organizations
+""""""""""""""""""""""""
+* Each user is assigned an individual organization with ``username`` as the organization name.
+* The user is the only member of this organization but may :ref:`invite collaborators <how-to_team-management_invite-to-project>` to projects.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 3]
```diff
@@ -1,116 +1,164 @@
...
+Roles in a project
+^^^^^^^^^^^^^^^^^^
+
+Members of projects can be one of three types: Owner, Contributor or Viewer.
+
+.. _core-concepts_user-roles_project-owner:
+
+**Owner**
+
+* Has edit access to all experiments and Notebooks.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 4]
```diff
@@ -1,116 +1,164 @@
...
+* Has edit access to project settings.
+* Can remove projects.
+* A project creator is by default the project Owner.
+
+**Contributor**
+
+* Has edit access to all experiments and Notebooks.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 5]
```diff
@@ -1,116 +1,164 @@
...
+* Has edit access to project settings.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 6]
```diff
@@ -1,116 +1,164 @@
...
+* Can run experiments.
+* Can add Notebooks and make checkpoints.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 7]
```diff
@@ -1,116 +1,164 @@
...
+* In the Settings tab, has view-only access to people in a project.
+* Can leave a project.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 8]
```diff
@@ -1,116 +1,164 @@
...
+
+**Viewer**
+
+* Has view-only access to experiments, Notebooks and wikis.
+* Cannot run experiments or make Notebook checkpoints.
+* Has no access to project settings.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/core-concepts.rst]: done

File: docs/learn-about-neptune/team-management.rst [hunk 9]
```diff
@@ -8,100 +8,103 @@ Team, project and user management
...
-Users will be invited via email to join your organization.
+- The tab displays the list of users who already belong to the organization or whose membership is pending.
+- Above the list is a field in which you can type an email or username  of people you want to invite. Neptune identifies names and emails for people who already have an account.
+
+3. Specify a name, set a role (Admin or Member) and click **Invite**.
 
-Create project
---------------
-How to create a new project?
+The users you invite will be invited via email to join your organization. Until they join, their status is marked as "Pending invitation".
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/team-management.rst]: done

File: docs/learn-about-neptune/team-management.rst [hunk 10]
```diff
@@ -8,100 +8,103 @@ Team, project and user management
...
    :target: ../_static/images/how-to/team-management/add-user-1.png
    :alt: Add users to project
 
-| If it is project created in team organization, user is assigned to project immediately.
-| If it is project created in the individual account, user is invited to the project via mail.
-  User needs to accept invite, before you can see such person on the list above.
+
+.. note::
+
+    - If the project is created in a team organization, the user is assigned to the project immediately and notified by email.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/team-management.rst]: I wrote:
The user is invited to the project by email. The user needs to accept the invitation, before that person appears in the list.

File: docs/learn-about-neptune/team-management.rst [hunk 11]
```diff
@@ -8,100 +8,103 @@ Team, project and user management
...
+    - If the project is created in an individual account, you can only add users to a public project. The user is notified by email and must accept the invitation to be added.
+    You cannot add users to a private project.
```
[the remark this answers is not in the document]
Comment (moshekruger) [reply docs/learn-about-neptune/team-management.rst]: Removed

File: docs/python-api/how-to/track.rst (hunk 1 above)
Comment (moshekruger) [reply docs/python-api/how-to/track.rst]: done

File: docs/learn-about-neptune/core-concepts.rst [hunk 12]
```diff
@@ -1,107 +1,121 @@
...
-
-* Has view-only access to experiments, notebooks and Wiki.
-* Cannot run experiments, make notebook checkpoints.
-* Has no access to project settings.
+^ Depends on organization type. Can leave public project in individual organization; cannot leave project in team organization.
 
 ====
 
-Organization types
+Organization Types
```
Comment (aniezurawski) [inline docs/learn-about-neptune/core-concepts.rst]: It should be at the top of this page, shouldn't it?

File: docs/learn-about-neptune/team-management.rst [hunk 13]
```diff
@@ -105,6 +106,5 @@ Add users to a project
 
 .. note::
 
-    - If the project is created in a team organization, the user is assigned to the project immediately and notified by email.
-    - If the project is created in an individual account, you can only add users to a public project. The user is notified by email and must accept the invitation to be added.
-    You cannot add users to a private project.
\ No newline at end of file
+    - The user is invited to the project by email. The user needs to accept the invitation, before that user appears in the list.
+    - If the project is created in an individual account, you can only add users to a public project. The user is notified by email and must accept the invitation to be added. You cannot add users to a private project.
```
Comment (aniezurawski) [inline docs/learn-about-neptune/team-management.rst]: you can only add users -> you can only invite users
You cannot add users -> You cannot invite users

File: docs/learn-about-neptune/team-management.rst [hunk 14]
```diff
@@ -105,6 +106,5 @@ Add users to a project
 
 .. note::
 
-    - If the project is created in a team organization, the user is assigned to the project immediately and notified by email.
-    - If the project is created in an individual account, you can only add users to a public project. The user is notified by email and must accept the invitation to be added.
-    You cannot add users to a private project.
\ No newline at end of file
+    - The user is invited to the project by email. The user needs to accept the invitation, before that user appears in the list.
```
Comment (aniezurawski) [inline docs/learn-about-neptune/team-management.rst]: by email or by username

File: docs/learn-about-neptune/team-management.rst [hunk 15]
```diff
@@ -105,6 +106,5 @@ Add users to a project
 
 .. note::
 
-    - If the project is created in a team organization, the user is assigned to the project immediately and notified by email.
```
Comment (aniezurawski) [inline docs/learn-about-neptune/team-management.rst]: We can keep this line. But I would move it below.

**Comment by moshekruger** [comment]
Great. Thanks.

----------------------
Moshe Kruger
Mobile: +972-52-770-7387
---------------------------------------
LinkedIn Profile: www.linkedin.com/in/allwrite

**Pull request merged**